src/main/kotlin/org/rust/ide/actions/diagnostic/CreateNewGithubIssue.kt

```diff
@@ -95,6 +108,17 @@ class CreateNewGithubIssue : DumbAwareAction() {
                 }
             }
 
+        private val additionalExperimentalFeatures: List<String>
+            get() = with(Experiments.getInstance()) {
+                listOf(
```

Comment:
The idea is to provide only disabled by default features to keep the issue description short.

I don't really like the fact that this list should be maintained and updated e.g. after we change `percentOfUsers` in XML or add a new experimental feature. However, I don't see a better way of doing it.

Reply:
Thanks for suggestion, I added an annotation